lib/os/Kconfig

```
@@ -157,6 +157,14 @@ config UTF8
 	  Enable the utf8 API. The API implements functions to specifically
 	  handle UTF-8 encoded strings.
 
+config MEM32
+	bool "Enable memcpy32()"
```

"Enable" is redundant.


```suggestion
	bool "memcpy32"
```